Configuration/TypoScript/setup.typoscript

```diff
@@ -0,0 +1,8 @@
+# Only include this TypoScript if the context matches (using # as the delimiter)
+[applicationContext matches "#^Development/BootstrapGrids/Docker#"]
+    # Enable errors to be displayed in the frontend
```

Comment:
I'd add this to the `Contribution/setup.typoscript` which is only included if runing in docker-dev-environment.

Reply:
Ahh ok i see your removed it from ext_localconf.php. Will just have a small look at our current file-orga, as we have now following:
```
Backend/setup.typoscript
Frontend/setup.typoscript
Contribution/setup.typoscript
setup.typoscript
```
Maybe i can organize this a bit better. Or maybe i will do that in a follow-up PR.

Reply:
I'm okay with whatever you decide. You can just add a commit to this branch if you want.

Reply:
👍🏽 

I now added the lowlevel extension here for debugging purposes in dev-environment.

Will now merge this. For me it is fine now.